src/NHibernate/Util/StringHelper.cs

```diff
@@ -83,11 +83,15 @@ public static string Repeat(string str, int times)
 			return buf.ToString();
 		}
 
+		//TODO: Inline
 		public static string Replace(string template, string placeholder, string replacement)
```

Comment:
Why not obsoleting and inlining it right now?

Reply:
Did not want to change much of the code.

Reply:
Well, if not now, why later? Better even remove the TODO then.

Reply:
TODO removed